core/utils/android-common/src/main/java/com/appcoins/wallet/core/utils/android_common/CurrencyFormatUtils.kt

```diff
@@ -42,6 +46,29 @@ class CurrencyFormatUtils @Inject constructor() {
     return formatCurrencyFiat(value)
   }
 
+  fun formatCost(currencyCode: String, currencySymbol: String, cost: BigDecimal?) : String {
```

Comment:
Please test with Mexico currency (to see if the currency symbol has the country letters associated with the $)